site/docs/data.md

```diff
@@ -71,6 +71,34 @@ For example, to create a data source named `myData`, use the following data
 }
 ```
 
+Then you can load it with vegaEmbed modying the [Vega View](https://vega.github.io/vega/docs/api/view/#data) using the insert or change methods.
```

Comment:
The Vega view also supports changesets.

Reply:
Can we write this less specifically about Vega-Embed? Something along the lines of "You can use the Vega view API to load data at runtime and update the chart. Here is an example using Vega-Embed:"

Reply:
I think it is better to explain it first without the changeset, as the code is simpler. I also agree on the rewording